Userland/Libraries/LibWeb/CSS/StyleValues/CalculatedStyleValue.h

```diff
@@ -313,4 +323,22 @@ class ClampCalculationNode final : public CalculationNode {
     NonnullOwnPtr<CalculationNode> m_max_value;
 };
 
+class SinCalculationNode final : public CalculationNode {
+public:
+    static ErrorOr<NonnullOwnPtr<SinCalculationNode>> create(NonnullOwnPtr<CalculationNode>);
+    ~SinCalculationNode();
+
+    virtual ErrorOr<String> to_string() const override;
+    virtual Optional<CalculatedStyleValue::ResolvedType> resolved_type() const override;
+    virtual bool contains_percentage() const override { return false; };
```

Comment:
As I've commented on some of the other PRs, this needs to check if the contained value has a percentage in it.